larch.js

```diff
@@ -41,8 +41,12 @@ function Larch(options) {
 
     if (self.backends.length === 1) {
         self.log = self.logSingleBackend;
```

Comment:
It's highly unlikely there will ever be more than 1 instance of Larch so I'm not super concerned about having different hidden classes